src/modules/dashboard/components/Dialogs/AwardAndSmiteDialogs/ManageReputationContainer/ManageReputationContainer.tsx

```diff
@@ -48,9 +49,12 @@ const ManageReputationContainer = ({
   const [userReputation, setUserReputation] = useState(0);
   const history = useHistory();
 
-  const { data: colonyMembers } = useMembersSubscription({
-    variables: { colonyAddress },
-  });
+  const { subscribedUsers } = colonyMembers;
+
+  const verifiedUsers = useMemo(() => {
+    const verifiedAddresses = colony.whitelistedAddresses;
+    return getVerifiedUsers(verifiedAddresses, subscribedUsers);
```

Comment:
You can just pass `colony.whitelistedAddresses` to the function, there is no need to create a variable.